kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMDocumentTest.java

```diff
@@ -91,40 +91,40 @@ public void that_FOR_PREVIEW_useSystemMenuBarProperty_is_enabled() throws Except
         FXOMObject fxomObject = classUnderTest.searchWithFxId("theMenuBar");
 
         assertTrue(fxomObject.getSceneGraphObject() instanceof MenuBar);
-        assertTrue("for preview, useSystemMenu is expected to be enabled",
-                ((MenuBar) fxomObject.getSceneGraphObject()).useSystemMenuBarProperty().get());
+        assertTrue(((MenuBar) fxomObject.getSceneGraphObject()).useSystemMenuBarProperty().get(),
+                "for preview, useSystemMenu is expected to be enabled");
 
         String generatedFxml = classUnderTest.getFxmlText(false);
         assertTrue(generatedFxml.contains("useSystemMenuBar=\"true\""));
     }
 
     @Test
     public void that_useSystemMenuBarProperty_is_disabled_on_MacOS() throws Exception {
-        assumeTrue("MacOS", OS.get() == OS.MAC);
+        assumeTrue(OS.get() == OS.MAC, "MacOS");
         
         classUnderTest = new FXOMDocument(fxmlText, fxmlUrl, loader, resourceBundle);
 
         FXOMObject fxomObject = classUnderTest.searchWithFxId("theMenuBar");
 
         assertTrue(fxomObject.getSceneGraphObject() instanceof MenuBar);
-        assertFalse("for preview, useSystemMenu is expected to be enabled",
-                ((MenuBar) fxomObject.getSceneGraphObject()).useSystemMenuBarProperty().get());
+        assertFalse(((MenuBar) fxomObject.getSceneGraphObject()).useSystemMenuBarProperty().get(),
+                "for preview, useSystemMenu is expected to be enabled");
 
         String generatedFxml = classUnderTest.getFxmlText(false);
         assertTrue(generatedFxml.contains("useSystemMenuBar=\"true\""));
     }
 
     @Test
     public void that_useSystemMenuBarProperty_not_modified_on_Linux_and_Windows() throws Exception {
-        assumeTrue("Windows or Linux", Set.of(OS.LINUX, OS.WINDOWS).contains(OS.get()));
+        assumeTrue(Set.of(OS.LINUX, OS.WINDOWS).contains(OS.get()), "Windows or Linux");
```

Comment:
Perhaps adding [disabled on os](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/condition/DisabledOnOs.html) or enabled on os would be clearer?

Reply:
Yes, very good idea. Much better than this.